### PR TITLE
Update readme

### LINKED ISSUE #37: Add downloads badge
Add below downloads badge in readme with build badge

```
[![NPM Downloads](https://img.shields.io/npm/dt/ak-phrase.svg)](https://www.npmjs.com/package/ak-phrase)
```

### PR DESCRIPTION
**Pull request by ghost**
Fixes #37 Added the download badge to the readme file

### CODE REVIEW

**Comment by ghost** [comment]
Feedback welcome

File: README.md [hunk 1]
```diff
@@ -22,11 +23,7 @@ $ npm install ak-phrasae.js
 ```js
 const sentenceGenerator = require('ak-phrase.js');
 
-const wordArray = [
-    ['eat'],
-    ['code', 'commit'],
-    ['sleep']
-];
+const wordArray = [['eat'], ['code', 'commit'], ['sleep']];
```
Comment (arshadkazmi42) [inline README.md]: Any specific reason for this change?
I was kept with line break intentionally, to make it easily readable.

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks good.
I have added a small feedback

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Oh no sorry while updating the ReadMe I had made a few accidental changes in the formatting but never reverted the changes my apologies there isn't any specific reason

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great. Just a small change. Can you also remove package-lock.json file from the PR.
And we will get this merged

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Ok nice

**Comment by ghost** [comment]
Ok I have made the necessary  changes to the repository

**Review by arshadkazmi42 — APPROVED** [review]
:+1:

**Comment by ghost** [comment]
Thank you I was kinda nice

**Pull request merged**